pom.xml

```diff
@@ -22,7 +22,7 @@
   <properties>
     <revision>1.5</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.161</jenkins.version>
```

Comment:
is this bump needed? what new code are you relying on?

Reply:
see: jenkinsci/jenkins#3817 (which I linked in the PR description).

Reply:
I see, this line:
https://github.com/jenkinsci/configuration-as-code-plugin/pull/683/files#diff-ee3d77b9c27bec86230982ed88cb8226R46

That's quite a big jump, not sure if this would be merged in a hurry

Reply:
I would guess it would need to be at least in an LTS version for this to be merged
@ewelinawilkosz ?